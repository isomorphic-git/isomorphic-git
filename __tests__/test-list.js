import git from '..'
import pify from 'pify'
import ncp from 'ncp'
import { tmpdir } from './__helpers__'

describe('list', () => {
  test('list', async () => {
    let dir = await tmpdir()
    await pify(ncp)('__tests__/__fixtures__/test-list.git', dir)
    const files = await git()
      .gitdir(dir)
      .list()
    expect(files).toMatchSnapshot()
  })
})
