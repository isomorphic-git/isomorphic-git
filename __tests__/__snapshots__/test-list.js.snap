// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list list 1`] = `
Array [
  ".babelrc",
  ".editorconfig",
  ".flowconfig",
  ".gitignore",
  ".travis.yml",
  "LICENSE.md",
  "README.md",
  "package-lock.json",
  "package.json",
  "shrinkwrap.yaml",
  "src/commands/checkout.js",
  "src/commands/config.js",
  "src/commands/fetch.js",
  "src/commands/init.js",
  "src/index.js",
  "src/models/GitBlob.js",
  "src/models/GitCommit.js",
  "src/models/GitConfig.js",
  "src/models/GitObject.js",
  "src/models/GitTree.js",
  "src/utils/exists.js",
  "src/utils/mkdirs.js",
  "src/utils/read.js",
  "src/utils/resolveRef.js",
  "src/utils/write.js",
  "test/_helpers.js",
  "test/snapshots/test-resolveRef.js.md",
  "test/snapshots/test-resolveRef.js.snap",
  "test/test-clone.js",
  "test/test-config.js",
  "test/test-init.js",
  "test/test-resolveRef.js",
]
`;
